content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -157,8 +157,8 @@ Initially, the pods are laid out as follows:
 | pod-a  *available*   | pod-b *available*   | pod-c *available*  |
 | pod-x  *available*   |                     |                    |
 
-All 3 pods are part of a deployment, and they collectively have a PDB which requires
-there be at least 2 of the 3 pods to be available at all times.
+The 3 pods (`pod-a`, `pod-b`, and `pod-c`) are part of a deployment, and they collectively have a PDB which requires
+there be at least 2 of the 3 pods for each to be available at all times.
```

Comment:
```suggestion
All 3 pods (`pod-a`, `pod-b`, and `pod-c`) are part of a deployment, and they collectively have a PDB which requires
there be at least 2 of the 3 pods to be available all times.
```

Reply:
@liviux ,
Do you agree with the suggestion?

Reply:
I'd capitalize the first letter of Deployment, and I'd write PodDisruptionBudget out longhand.

Also, I prefer “that requires” over “which requires”.

@kundan2707 is your suggestion the same as skipping this PR? If not, what did I miss?